packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts

```diff
@@ -108,17 +108,13 @@ export class PhoenixMenuNode {
 
   /**
    * Add a config to the phoenix menu item.
-   * @param type Type of configuration.
-   * @param options Options for the config.
+   * @param config config to be displayed as a Phoenix Menu item.
    * @returns The current node.
    */
-  addConfig<T extends keyof PhoenixMenuConfigs>(
```

Comment:
So, the plan here is to avoid type inference and make it explicit in the caller? Is there an advantage to this i.e. was this the cause of the problem?

(Just for my understanding)

Reply:
ping @emiliocortina ?